src/renate/updaters/model_updater.py

```diff
@@ -411,8 +411,8 @@ def _fit_learner(
 
         strategy = create_strategy(self._devices, self._strategy)
         # Finetuning-equivalent epochs.
-        num_batches = len(learner._train_dataset) // learner._batch_size
-        num_batches += min(len(learner._train_dataset) % learner._batch_size, 1)
+        num_batches = len(learner._train_dataset) // 256  # learner._batch_size
+        num_batches += min(len(learner._train_dataset) % 256, 1)  # learner._batch_size, 1)
```

Comment:
not sure why this doesn't work on github, but it works locally. the reason the tests are failing is that a pass over the data has now effectively fewer batches.

what's a bit concerning is that I saw that the validation loss has changed. are there any changes to how the validation buffer gets populated or do we drop the last batch of the validation buffer? funnily, that does not happen for each epoch.